src/dialogs/linkdialog.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Search note</string>
```

Comment:
Oh, I never saw that one because I overwrite the title in source code. Thank you!